### PR TITLE
Add team members, prescribed project readme structure

### PR DESCRIPTION
**Pull request by wayneadams**

### CODE REVIEW

**Comment by wayneadams** [comment]
resolves #47

File: README.md [hunk 1]
```diff
@@ -7,12 +7,58 @@ Divvying up expenses between friends and peers can be a disaster. Remember that
 
 Divvy allows users to enter expenses as they are incurred, and calculates the best way to settle the bill. No more painful math and messy payments to different people, Divvy simplifies everything down to one payment per person.
 
+## Team
 
+  - __Product Owner__: [Hannah Henderson](https://github.com/hchsquared)
+  - __Scrum Master__: [James Ramadan](https://github.com/jamesramadan)
+  - __Development Team Members__: [Dennis Ting](https://github.com/dting), [Wayne Adams](https://github.com/wayneadams)
```
Comment (dting) [inline README.md:14]: Do we want to include all the names in this category??

**Comment by jamesramadan** [comment]
I noticed we included PostGres within the "Requirements" section. Does this line apply to our project? Aren't we using another RDBMS?

**Comment by dting** [comment]
Thanks, Wayne this LGTM. I noticed the same thing as James. Let's throw up an issue to get that fixed at some point.

**Comment by jamesramadan** [comment]
Second Dting. Everything else looks good.

File: README.md (hunk 1 above)
Comment (wayneadams) [inline README.md:14]: @dting, I think they're looking for our README.md to have all of our members

**Comment by wayneadams** [comment]
@jamesramadan @dting yeap, everything from usage down needs to be updated later

**Pull request merged**